go-controller/pkg/libovsdbops/db_object_types.go

```diff
@@ -158,3 +161,11 @@ var ACLEgressFirewall = newObjectIDsType(acl, EgressFirewallOwnerType, []Externa
 	// The only additional id we need is the index of the EgressFirewall.Spec.Egress rule.
 	RuleIndex,
 })
+
+var VirtualMachineDHCPOptions = newObjectIDsType(dhcpOptions, VirtualMachineOwnerType, []ExternalIDKey{
```

Comment:
We can do that but it is not mandatory. I don't think we have really decided if we should add everything we can here or not. But if it is a must because we are concerned about performance with this specific table, then definitely we should add it here.

Reply:
I guess why not? Let's add it

Reply:
So I should add DHCPOptions here similar to `LoadBalancerTable` and remove the need for predicates at the helper functions ?

Reply:
Yup although it would be more akin to the `ACLTable` not the `LoadBalancerTable`